apps/io-wallet-user-func/src/infra/azure/functions/add-wallet-instance-to-validation-queue.ts

```diff
@@ -0,0 +1,19 @@
+import { AddWalletInstanceToValidationQueueHandler } from "@/infra/handlers/add-wallet-instance-to-validation-queue";
+import { azureFunction } from "@pagopa/handler-kit-azure-func";
+import { pipe } from "fp-ts/function";
+import * as RTE from "fp-ts/lib/ReaderTaskEither";
+import { sendTelemetryException } from "io-wallet-common/infra/azure/appinsights/telemetry";
+
+export const AddWalletInstanceToValidationQueueFunction = azureFunction(
+  pipe(
+    AddWalletInstanceToValidationQueueHandler,
+    RTE.orElseFirstW((error) =>
```

Comment:
for consistency with the other functions, we should place the telemetry part in the handlers